include/manif/impl/rn/RnTangent_base.h

```diff
@@ -202,6 +202,7 @@ struct RandomEvaluatorImpl<RnTangentBase<Derived>>
   }
 };
 
+//! @brief Bracket specialization for RnTangentBase objects.
```

Comment:
Does this bracket thing belong to this PR?

Reply:
Yeah it's just a small comment/doc that's missing and which I stuff in this pr.